app/controllers/mentee_profiles_controller.rb

```diff
@@ -1,6 +1,29 @@
 class MenteeProfilesController < ApplicationController
-	def show
-	end
-	def dashboard
-	end
+  before_action :require_mentee, except: [ :show ]
+  before_action :require_mentee_or_organiser, only: [:show]
+
+  def show
+    @user = User.find(params[:id])
+    @mentor = User.find(@user.matched_id)
+
+  end
+  
+  def dashboard
+    @mentor = User.find(current_user.matched_id)
+  end
+  
+  def missing_mentor
+    @user = current_user
+    if !@user.blank? 
```

Comment:
you've added before_action so there's no way user will ever be blank at this point